src/Command/ProjectBuildCommand.php

```diff
@@ -2,311 +2,320 @@
+        $platform_routes['https://www.' . $site['url'] . '/'] = [
+          'type' => 'upstream',
+          'upstream' => $platform['name'] . ':http',
+          'cache' => [
+            'enabled' => 'false',
+          ],
         ];
 
-        // Attempt to write the YAML configuration files.
-        foreach ($config_files as $file => $file_data) {
-            try {
-                $filesystem->dumpFile(getcwd() . $file, $file_data[0]);
-                $io->success("Created $file_data[1] file");
-            }
-            catch (IOExceptionInterface $exception) {
-                $io->error("Unable to create $file_data[1] file, error: " . $exception->getMessage());
-            }
-        }
-
-        // Copy Platform Solr server configuration.
-        try {
-            $filesystem->mirror(getcwd() . '/.hosting/platformsh/solr_config', getcwd() . '/.platform/solr_config');
-            $io->success('Successfully copied Solr server configuration');
-        }
-        catch (IOExceptionInterface $exception) {
-            $io->error('Unable to copy Solr server configuration, error: ' . $exception->getMessage());
+        $platform_routes['https://' . $site['url'] . '/'] = [
+          'type' => 'redirect',
+          'to' => 'https://www.' . $site['url'] . '/',
+        ];
+      }
+
+      // If a site folder doesn't exist under project/sites, create it and
+      // provide a settings file.
+      if (!$this->fs()->exists('/project/sites/' . $site_id)) {
+        $io->text('Creating a site directory for ' . $site_id . ' under project/sites/');
+        $this->fs()->mkdir('/project/sites/' . $site_id);
+        $this->fs()->copy('/.lando/config/multisite.settings.php', '/project/sites/' . $site_id . '/settings.php');
+      }
+
+      // Enable our multisite entry by linking from the sites directory to
+      // the project directory.
+      try {
+        $this->fs()->symlink('//app/project/sites/' . $site_id, '/web/sites/' . $site_id);
+        $io->text('Linking sites directory');
+      }
+      catch (IOExceptionInterface $exception) {
+        $io->error("An error occurred while linking $site_id site directory: " . $exception->getMessage());
+      }
+
+      // If a site config doesn't exist under project/config, create it.
+      if (!$this->fs()->exists('/project/config/' . $site_id)) {
+        $io->text('Creating config directory for ' . $site_id . ' under project/config/');
+        $this->fs()->mkdir('/project/config/' . $site_id);
+        $this->fs()->dumpFile('/project/config/' . $site_id . '/.gitkeep', "");
+
+        // Create the default config directories if they don't already exist.
+        foreach (['config', 'hosted', 'local', 'production'] as $directory) {
+          $io->text('Creating default config directories');
+          if (!!$this->fs()->exists('/project/config/' . $site_id . '/config/' . $directory)) {
+            $this->createDirectory('/project/config/' . $site_id . '/config/' . $directory);
```

Comment:
This looks wrong, I think that this is why we still have project/config/<sitenam>/config directories being created when we don't want that. Shouldn't it be:
$this->createDirectory('/project/config/' . $site_id . '/' . $directory);

Reply:
You are quite right, good spot.